### PR TITLE
Add buid telemetry notes to tools packages

### PR DESCRIPTION
**Pull request by pepone**
This PR adds build telemetry section to the Protobuf and Slice tools packages.

### CODE REVIEW

File: tools/IceRpc.Protobuf.Tools/README.md [hunk 1]
```diff
@@ -87,6 +87,34 @@ This package includes the `protoc` compiler binaries, and the Protobuf well-know
...
+This data includes:
+
+- The IceRPC version.
+- The system's operating system, version, and platform architecture.
+- The source of the telemetry data (`IceRpc.Protobuf.Tools`).
+- The number of Protobuf files being compiled.
+- A SHA256 hash computed from the Protobuf files being compiled.
+
+This data is used to help the IceRPC team understand how the tools are being used and to prioritize future development
+efforts. The data is stored in a secure database and is not shared with any third parties.
```
Comment (externl) [inline tools/IceRpc.Protobuf.Tools/README.md:106]:
```suggestion
efforts. The data is stored in a private database and is not shared with any third parties.
```

File: tools/IceRpc.Slice.Tools/README.md [hunk 2]
```diff
@@ -96,14 +96,45 @@ unique set of options results in a separate execution of `slicec-cs`.
...
+- The system's operating system, version, and platform architecture.
+- The source of the telemetry data (`IceRpc.Slice.Tools`).
+- The number of Slice files being compiled.
+- The number of Slice reference files.
+- Whether or not the build contains Slice1 definitions.
+- Whether or not the build contains Slice2 definitions.
+- A SHA256 hash computed from the Slice files being compiled.
+
+This data is used to help the IceRPC team understand how the tools are being used and to prioritize future development
+efforts. The data is stored in a secure database and is not shared with any third parties.
```
Comment (externl) [inline tools/IceRpc.Slice.Tools/README.md:126]:
```suggestion
efforts. The data is stored in a private database and is not shared with any third parties.
```

**Review by externl — APPROVED** [review]
Looks good. Slightly related to this we should really have an env variable that ca be used to disable telemetry if we don't already have it.

File: tools/IceRpc.Protobuf.Tools/README.md [hunk 3]
```diff
@@ -87,6 +87,34 @@ This package includes the `protoc` compiler binaries, and the Protobuf well-know
 [Google.Protobuf.Tools][google-protobuf-tools] package. Additionally, it includes the `protoc` ARM64 binaries
 for Linux and macOS from the [Google Protobuf release page][google-protobuf-release].
 
+## Build Telemetry
```
Comment (bernardnormier) [inline tools/IceRpc.Protobuf.Tools/README.md:90]: Should be "Build telemetry" - same capitalization as other `##` headings.

File: tools/IceRpc.Slice.Tools/README.md [hunk 4]
```diff
@@ -96,14 +96,45 @@ unique set of options results in a separate execution of `slicec-cs`.
...
-| `--rpc none`                        | [ZeroC.Slice][zeroc-slice]    |
+| Rpc metadata                  | NuGet package reference      |
+|-------------------------------|------------------------------|
+| `icerpc` or no `Rpc` metadata | [IceRpc.Slice][icerpc-slice] |
+| `none`                        | [ZeroC.Slice][zeroc-slice]   |
 
 Referencing `IceRpc.Slice` makes your project reference transitively [ZeroC.Slice][zeroc-slice], [IceRpc][icerpc] and
 [System.IO.Pipelines][system-io-pipelines].
 
+## Build Telemetry
```
Comment (bernardnormier) [inline tools/IceRpc.Slice.Tools/README.md:107]: Build telemetry

**Pull request merged**